pkg/infra/models/jira_customFields.go

```diff
@@ -7,6 +7,22 @@ import (
 
 type CustomFields struct{ Fields []map[string]interface{} }
 
+// In order to clear a field create an emtpy field which will wipe the value
+func (c *CustomFields) EmptyField(customFieldID string) (err error) {
+	if len(customFieldID) == 0 {
+		return fmt.Errorf("error, please provide a valid customFieldID value")
+	}
+
+	var fieldNode = map[string]interface{}{}
+	fieldNode[customFieldID] = nil
+
+	var fieldsNode = map[string]interface{}{}
+	fieldsNode["fields"] = fieldNode
+
+	c.Fields = append(c.Fields, fieldsNode)
```

Comment:
maybe its me but my brain is struggling to parse this, maybe
```suggestion
	c.Fields = append(c.Fields, map[string]interface{}{
		"fields": map[string]interface{}{
			[customFieldID]: nil,
		}
	})
```
not fussed though

Reply:
Oh I was just copying the weird style in the rest of the file haha - this does make it more readable yeah